{
    "cve": {
        "data_type": "CVE",
        "data_format": "MITRE",
        "data_version": "4.0",
        "CVE_data_meta": {
            "ID": "CVE-2020-25032",
            "ASSIGNER": "cve@mitre.org"
        },
        "problemtype": {
            "problemtype_data": [
                {
                    "description": []
                }
            ]
        },
        "references": {
            "reference_data": [
                {
                    "url": "https://github.com/corydolphin/flask-cors/releases/tag/3.0.9",
                    "name": "https://github.com/corydolphin/flask-cors/releases/tag/3.0.9",
                    "refsource": "MISC",
                    "tags": []
                }
            ]
        },
        "description": {
            "description_data": [
                {
                    "lang": "en",
                    "value": "An issue was discovered in Flask-CORS (aka CORS Middleware for Flask) before 3.0.9. It allows ../ directory traversal to access private resources because resource matching does not ensure that pathnames are in a canonical format."
                }
            ]
        }
    },
    "configurations": {
        "CVE_data_version": "4.0",
        "nodes": []
    },
    "impact": {},
    "publishedDate": "2020-08-31T04:15Z",
    "lastModifiedDate": "2020-08-31T04:15Z"
}